filament/src/components/RenderableManager.cpp

```diff
@@ -372,6 +374,16 @@ void FRenderableManager::create(
                 }
             }
         }
+
+        if (builder->mMorphingEnabled) {
+            MorphWeights& morphWeights = manager[ci].morphWeights;
+            morphWeights = MorphWeights {
+                .handle = driver.createBufferObject(
```

Comment:
add a comment (same as above) that we could use a single UBO for several primitives and use `bindUniformBufferRange`, which might be more efficient.

Reply:
It's done.